ang/civicase/Search.js

```diff
@@ -79,6 +91,16 @@
       });
     });
 
+    /**
+     * Watcher for relationshipType filter
+     */
+    $scope.$watch('relationshipType', function () {
```

Comment:
Please follow the following kind of convention for all watchers.
https://github.com/compucorp/uk.co.compucorp.civicase/blob/5064ec447545eee3ac64209576c39b5c24498450/ang/civicase/CaseDetails.js#L39